pkg/gnmi/util.go

```diff
@@ -324,3 +333,242 @@ func setPathWithoutAttribute(op pb.UpdateResult_Operation, curNode map[string]in
 	}
 	return nil
 }
+
+// sendResponse sends an SubscribeResponse to a gNMI client.
+func (s *Server) sendResponse(response *pb.SubscribeResponse, stream pb.GNMI_SubscribeServer) {
+	log.Info("Sending SubscribeResponse out to gNMI client: ", response)
+	err := stream.Send(response)
+	if err != nil {
+		//TODO remove channel registrations
+		log.Errorf("Error in sending response to client %v", err)
+	}
+}
+
+// buildSubResponse builds a subscribeResponse based on the given Update message.
+func buildSubResponse(update *pb.Update) (*pb.SubscribeResponse, error) {
```

Comment:
please see the update https://github.com/onosproject/onos-config/pull/373 in ONOS-config to handle deletes